node/src/components/consensus/consensus_protocol.rs

```diff
@@ -93,6 +93,7 @@ pub(crate) enum ProtocolOutcome<I, C: Context> {
     CreateNewBlock {
         block_context: BlockContext,
         past_values: Vec<C::ConsensusValue>,
+        parent_value: Option<C::ConsensusValue>,
```

Comment:
Why not make it part of the `BlockContext`?

Reply:
I'm trying, but this turns out to be a major change: We'd have to make `BlockContext` (which is created inside `highway_core`) generic in `C: Context`, which causes other issues…
Since I'd like to rethink that solution before porting it to `dev` anyway, maybe it's not worth doing this in this PR?